lib/web/conn_upgrade_test.go

```diff
@@ -87,7 +87,7 @@ func TestHandlerConnectionUpgrade(t *testing.T) {
 		go func() {
 			// serverConn will be hijacked.
 			w := newResponseWriterHijacker(nil, serverConn)
-			_, err = h.connectionUpgrade(w, r, nil)
+			_, err := h.connectionUpgrade(w, r, nil)
```

Comment:
This was just to get unit tests to pass pending merge of a flaky test fix in another PR.